src/Formatter/CrossReferencesConfigurator.php

```diff
@@ -71,21 +72,22 @@ public function __invoke(Configurator $config): void
     public static function filterCrossReferences(Tag $tag, ?User $actor): bool
     {
         if (is_null($actor)) {
-            $log = resolve(LogReporter::class);
-            $msg = 'actor is "null", falling back to invalidating tag. This is probably due to another extension not passing this parameter to "Formatter->parse()". See stack trace below.';
-            $log->report(new RuntimeException($msg));
-            $tag->invalidate();
-            return false;
+            if (resolve('flarum.config')->inDebugMode()) {
```

Comment:
Why only log the error in development mode? How are supposed the admins to understand the issue if no error is logged? And what is the benefit of not logging it?

Reply:
In many cases this log message is irrelevant. In my case forum is public, it does not matter if extension can't detect actor, because fallback to guest will handle this correctly in 99%. and for the rest 1% displaying raw link is fine. I don't need a logs full of these messages, because I do not have plans to fix extensions that may cause it. And if someone wants to understand why links are not parsed, debug mode is the right way to do this.

Reply:
OK I guess, It is true that it adds a lot of noise in the logs. Maybe only logging one line in production mode instead of the full stack trace would be a good compromise? While keeping the stack trace in debug mode?